backend/.env.example

```diff
@@ -1 +1,2 @@
-DATABASE_URL='postgresql://<your_username>:<your_password>@localhost:5432/journaly_db'
+FRONTEND_URL='http://localhost:3000'
```

Comment:
I didn't have these 2 values in any .env file, but they were required to run the server, so I added them to the example file